lib/menu.rb

```diff
@@ -0,0 +1,16 @@
+class Menu
+
+  attr_reader :content, :see
+
+  def initialize
+    @content = { chicken: 4, lamb: 7, vegetables: 3, latte: 2 }
+    @see = proc { |x| puts (x.capitalize.to_s).ljust(10) + content[x].to_s.rjust(10) }
```

Comment:
Line is too long. [86/80]<br>Do not write to stdout. Use Rails' logger if you want to log.